tests/test_runner/test_parallel.py

```diff
@@ -44,6 +51,16 @@ def dummy_test(self):
 
 class RemoteTestResultTest(SimpleTestCase):
 
+    def test_pickle_errors_detection(self):
+        picklable_error = RuntimeError("This is fine")
+        not_unpicklable_error = ErrorThatFailsUnpickling("one arg", "another arg")
+
+        result = RemoteTestResult()
+        result._raise_when_not_picklable(picklable_error)
+
+        with self.assertRaisesMessage(TypeError, "argument"):
```

Comment:
Including a bit more of the message might help readability to understand exactly what `TypeError` is raised..